src/Symfony/Component/Console/Input/InputArgument.php

```diff
@@ -50,7 +50,7 @@ class InputArgument
      * @param string|bool|int|float|array|null                                              $default         The default value (for self::OPTIONAL mode only)
      * @param array|\Closure(CompletionInput,CompletionSuggestions):list<string|Suggestion> $suggestedValues The values used for input completion
      *
-     * @throws InvalidArgumentException When argument mode is not valid
+     * @throws InvalidArgumentException When argument name is 'command' or mode is not valid
```

Comment:
```suggestion
     * @throws InvalidArgumentException When argument name is "command" or mode is not valid
```

To be consistent with the rest

Reply:
Suggestion applied